src/lib.rs

```diff
@@ -254,7 +256,12 @@ mod candle_auction {
                 }
             }
 
-            // pay the looser his bidded amount back
+            // pay the loser his bid amount back
+            // TODO: shouldn't we handle multiple accounts here? I guess the
```

Comment:
Please see line  [415](https://github.com/agryaznov/candle-auction-ink/pull/1/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759L415): the payout is invoked by each participant by himself. Which is considered fair enough as winner payout presumably takes more gas fees to execute, hence everyone pays his own fees only